client/src/app/components/client-gridview/client-gridview.component.spec.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Please you need to test the initialization of the component.

Reply:
fixed in commit 784cde4613b4b1f0bc0728d793ac2f923919a2a9